tests/trading/test_completeSets.py

```diff
@@ -36,12 +36,12 @@ def test_publicBuyCompleteSets(contractsFixture):
         {
             "_event_type": "CompleteSets",
             "sender": bytesToHexString(tester.a1),
-	    "reportingFee": 0L,
+	        "reportingFee": 0L,
```

Comment:
Mixed tabs and spaces.  Consider using a `.editorconfig` extension to address this problem as you type.